TS/src/BenchmarkRunner.lf

```diff
@@ -0,0 +1,117 @@
+target TypeScript;
+
+/*
+ *
+ * Copyright (C) 2022 TU Dresden and UC Berkeley
+ *
+ * @author Wonseo Choi
+ */
+
+reactor BenchmarkRunner(num_iterations:number(12)) {
+
+    output start:boolean;
+    input finish:boolean;
+    
+    logical action nextIteration:boolean;
+    logical action done:boolean;
+    
+    state count:number(0);
+    
+    /** Start time for runtime measurement. */
+    state startTime:time;
+    
+    /** Runtime measurements. */
+    //TODO : "any" Array ! -> change another function code //
+    state measuredTimes: {=Array<any>=}({= [] =});
```

Comment:
Looks like the type should be `Array<number>`?

Reply:
Thank you for picking up the missing point!
I modify the function and little part to use Array<number>.
https://github.com/lf-lang/benchmarks-lingua-franca/pull/13/commits/0bb3a1de4e13358fc9a6e1067fa526f4ba0ca906